### ISSUE TITLE
[ui5-carousel]: [a11y] heading levels within carousel are not being announced by screenreader

### ISSUE THREAD

**Issue by I585157** [SAP/ui5-webcomponents#8624]
### Bug Description

Child items that either have a section heading tag, or an element with both role = "heading" and aria-level set do not have their heading level announced by a screen reader when placed in a carousel. 

### Affected Component

ui5-carousel

### Expected Behaviour

All items with a section heading tag or with role = "heading" and aria-level set should be announced as a header with their heading level by the screenreader. 

### Isolated Example

https://codesandbox.io/p/sandbox/laughing-browser-l72jfg?file=%2Fsrc%2FApp.js%3A18%2C18

### Steps to Reproduce

1. Open JAWS or NVDA screenreader and navigate to https://codesandbox.io/p/sandbox/laughing-browser-l72jfg?file=%2Fsrc%2FApp.js%3A18%2C18
2. if using JAWS press the h button to loop through all the headers. Verify that the headers within the carousel aren't read out. 
3. Then, tab to the carousel items. 
4. Verify that the carousel items aren't announced as headers or with a heading level
...

### Priority

None

### UI5 Web Components Version

1.23.0

### Browser

Chrome

### Operating System

Mac OS

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by dobrinyonkov** [comment]
Hello @SAP/ui5-webcomponents-topic-rd team,

Could you check this issue about the carousel and the headings not announced by screen readers (using PC cursor)?

Thanks and Best Regards,
Dobrin

**Comment by dimovpetar** [comment]
```html
```

**Comment by GerganaKremenska** [comment]
Hello @I585157, 

This issue will be handled with this backlog item - FIORITECHE1-8307 - Follow for updates here.

As this is not an issue, but needs an accessibility redesign i am closing the github issue. 

Best Regards,
Gergana
